bigchaindb/common/schema/transaction.yaml

```diff
@@ -105,13 +105,14 @@ definitions:
     description: |
         Description of the asset being transacted. In the case of a ``TRANSFER``
         transaction, this field contains only the ID of asset. In the case
-        of a ``CREATE`` transaction, this field may contain properties:
+        of a ``CREATE`` transaction, this field contains only a user payload
+        associated with the asset.
     additionalProperties: false
-    required:
-    - id
```

Comment:
Turns out there's a few other places where we're generalizing properties to fit different use cases (e.g. `CREATE`'s inputs).

This is a fairly different problem at hand, so we'll leave it for another issue: https://github.com/bigchaindb/bigchaindb/issues/946.